### PR TITLE
Adds migration procedure to Quay

### PR DESCRIPTION
**Pull request by stevsmit**
https://stevsmit.github.io/quay-docs/standalone-migration/html-single/#migrating_a_standalone_red_hat_quay_deployment_to_openshift_container_platform

### CODE REVIEW

File: modules/standalone-to-operator-backup-restore.adoc [hunk 1]
```diff
@@ -0,0 +1,382 @@
...
+. Copy the database SQL backup to the Quay PostgreSQL database instance:
++
+[source,terminal]
+----
+$ oc cp /tmp/user/quay-backup/quay-database-backup.sql quay-enterprise/quayregistry-quay-database-54956cdd54-p7b2w:/var/lib/pgsql/data/userdata
+----
++
+[NOTE]
+====
+In versions of OpenShift Container Platform earlier than 4.10, you must use the following command: `$ oc cp /tmp/quay-backup/quay-database-backup.sql quay-postgresql-database-pod:/var/lib/pgsql/data/userdata`.
```
Comment (hu-weihua) [inline modules/standalone-to-operator-backup-restore.adoc:152]: I have not tried all oc versions,  but I checked oc 4.6, 4.7, 4.8.  I got the same man page
```
$ ./oc cp --help
Copy files and directories to and from containers.

Usage:
  oc cp <file-spec-src> <file-spec-dest> [flags]

Examples:
........
  # Copy /tmp/foo local file to /tmp/bar in a remote pod in namespace <some-namespace>
  oc cp /tmp/foo <some-namespace>/<some-pod>:/tmp/bar
.....
```

File: modules/standalone-to-operator-backup-restore.adoc [hunk 2]
```diff
@@ -0,0 +1,382 @@
...
+$ oc scale --replicas=1 deployment quayregistry-quay-app
+deployment.apps/quayregistry-quay-app scaled
+----
+
+. Scale up the mirror pods:
++
+----
+$ oc scale --replicas=1  deployment quayregistry-quay-mirror
+deployment.apps/quayregistry-quay-mirror scaled
+----
```
Comment (hu-weihua) [inline modules/standalone-to-operator-backup-restore.adoc:322]: here need to scale operator pod
```
$ oc scale --replicas=1 deployment quay-operator.v3.6.2 -n openshift-operators
```

File: modules/standalone-to-operator-backup-restore.adoc [hunk 3]
```diff
@@ -0,0 +1,382 @@
...
+          port: "9000"
+          secret_key: ##########
+          storage_path: /datastorage/registry
+----
+
+. Store the `access_key` and `secret_key` from the Quay `config.yaml` file in your `~/.aws` directory:
++
+[source,terminal]
+----
+$ touch example_file
```
Comment (hu-weihua) [inline modules/standalone-to-operator-backup-restore.adoc:67]: What we want to run is below command, right?
```
$ touch ~/.aws/credentials
```

File: modules/standalone-to-operator-backup-restore.adoc [hunk 4]
```diff
@@ -0,0 +1,379 @@
...
++
+[source,terminal]
+----
+$ oc cp /tmp/user/quay-backup/quay-database-backup.sql quay-enterprise/quayregistry-quay-database-54956cdd54-p7b2w:/var/lib/pgsql/data/userdata
+----
++
+[NOTE]
+====
+In versions of OpenShift Container Platform earlier than 4.10, you must use the following format: `$ oc cp <file-spec-src> <file-spec-dest> [flags]`.
+====
```
Comment (hu-weihua) [inline modules/standalone-to-operator-backup-restore.adoc:150]: I think we don't need to add this `NOTE`.  In my last comment, I mean the `oc cp` command is the same in OCP 4.6, 4.7, 4.8. and 4.10.  I guess the other `oc` versions should be the same.  So the below one command is enough. 
```
----
$ oc cp /tmp/user/quay-backup/quay-database-backup.sql <quay-namespace>/<quay_pod_name>:/var/lib/pgsql/data/userdata
----
```

**Comment by huweihua-redhat** [comment]
LGTM, thanks @stevsmit

**Pull request merged**